keanu-project/src/main/java/io/improbable/keanu/tensor/Tensor.java

```diff
@@ -53,7 +53,7 @@ static BooleanTensor elementwiseEquals(Tensor a, Tensor b) {
 
     T getValue(int... index);
 
-    void setValue(T value, int... index);
+    Tensor<T> setValue(T value, int... index);
```

Comment:
Does this return the updated tensor? Did it just mutate the tensor before?

Reply:
Yep to both